src/site/_drafts/what-should-you-measure-to-improve-performance/index.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: What should you measure to improve performance
+author: martinschierle
+description: |
+  Learn what to measure to optimize different parts of the funnel for performance and therefore conversions.
+web_lighthouse: N/A
+date: 2019-05-01
+---
+
+
+The different steps of a purchase funnel are prone to performance issues in
+different ways, and therefore need different measurement and optimizations:
+
+![image](funnel.png)
+
+In this guide we'll address how the different steps can be measured. For this we
+recommend you to look at lab as well as field data.
+
+**Lab data** is gathered by running tests locally, for example by using
+[Lighthouse](https://web.dev/fast/discover-performance-opportunities-with-lighthouse)
+and other tools.  This can make it possible  to compare website performance over
+time and with competitors through a controlled, stable environment, but it might
+not be representative of the performance users experience in real life.   
+
+**Field data** is gathered via analytics from real users, and is therefore
```

Comment:
Not sure if it's relevant to the topic but wanted to mention there is a section on field data: https://web.dev/fast#measure-performance-in-the-field. Could maybe pop it into a note if you think it'd be interesting for folks.

Reply:
Added a sentence and link